Earthfile

```diff
@@ -290,29 +295,41 @@ framework:
 
     SAVE ARTIFACT --keep-own /framework/ framework
 
+multi-build-framework-image:
+    ARG --required SECUIRTY_PROFILE
+
+    BUILD --platform=linux/amd64 --platform=linux/arm64 +build-framework-image
+
 build-framework-image:
     FROM alpine
+    ARG SECURITY_PROFILE
 
-    ARG --required FRAMEWORK_FLAVOR
-
-    # Just in case, make sure this is valid
-    IF [ "$FRAMEWORK_FLAVOR" = "" ]
-        ARG FRAMEWORK_FLAVOR=generic
+    IF [ "$SECURITY_PROFILE" = "fips" ]
+        ARG _SECUIRTY_PROFILE=fips
+    ELSE
+        ARG _SECUIRTY_PROFILE=generic
     END
 
+
     COPY +version/VERSION ./
     ARG VERSION=$(cat VERSION)
 
-    ARG _IMG="$IMAGE_REPOSITORY_ORG/framework:${VERSION}_${FRAMEWORK_FLAVOR}"
+    IF [ "$VERSION" ~= ".*dirty.*" ]
+        ARG FRAMEWORK_VERSION=master
```

Comment:
we will probably revisit this when we start this: https://github.com/kairos-io/kairos/issues/1975 but shouldn't we let the caller pass the version down? Matching `dirty` feels...well, dirty :D 

Feel free to ignore for now, this will change on the #1975 anyway.

Reply:
yeah right now I thought there would be only 2 options, master or a release tag ... but indeed lets evaluate if there's a better way to do it